clj/src/slipstream/ui/models/dashboard.clj

```diff
@@ -17,17 +18,35 @@
   [cloud-usages]
   (cons
     {:cloud         (t :all-clouds.title)
+     :global-usage? true
      :current-usage (->> cloud-usages (map :current-usage) (reduce +))
      :quota         (->> cloud-usages (map :quota)         (reduce +))}
     cloud-usages))
 
+(defn- visible?
+  [cloud-usage]
+  (let [configured-clouds (into #{} (current-user/configuration :configured-clouds))]
+    (or
+      (-> cloud-usage :global-usage?)
+      (-> cloud-usage :current-usage pos?)
+      (-> cloud-usage :cloud configured-clouds))))
```

Comment:
Nope: using it within the `->` macro, the result of `:cloud` would be placed as the first parameter of `set`. But I've replaced the `(into #{} ...` with `(set ...` as you suggest, because its more readable and has the same effect: provide an empty set if `(current-user/configuration :configured-clouds)` is `nil`. I'm testing and pushing shortly. Thanks!